lib/nextstrain/sphinx/theme/footer.html

```diff
@@ -22,182 +22,87 @@
+    {{ TeamMember(name="James Hadfield", image="james-hadfield.jpg", link="http://bedford.io/team/james-hadfield/") }},
+    {{ TeamMember(name="Emma Hodcroft", image="emma-hodcroft.jpg", link="http://emmahodcroft.com/") }},
+    {{ TeamMember(name="Thomas Sibley", image="thomas-sibley.jpg", link="https://bedford.io/team/thomas-sibley/") }},
+    {{ TeamMember(name="John Huddleston", image="john-huddleston.jpg", link="http://bedford.io/team/john-huddleston/") }},
+    {{ TeamMember(name="Ivan Aksamentov", image="ivan-aksamentov.jpg", link="https://neherlab.org/ivan-aksamentov.html") }},
+    {{ TeamMember(name="Jover Lee", image="jover-lee.jpg", link="http://bedford.io/team/jover-lee/") }},
+    {{ TeamMember(name="Kairsten Fay", image="kairsten-fay.jpg", link="http://bedford.io/team/kairsten-fay/") }},
+    {{ TeamMember(name="Moira Zuber", image="moira-zuber.jpg") }},
+    {{ TeamMember(name="Eli Harkins", image="eli-harkins.jpg", link="https://bedford.io/team/eli-harkins/") }},
+    {{ TeamMember(name="Misja Ilcisin", image="misja-ilcisin.jpg", link="http://bedford.io/team/misja-ilcisin/") }},
+    {{ TeamMember(name="Cassia Wagner", image="cassia-wagner.jpg", link="https://bedford.io/team/cassia-wagner/") }},
+    {{ TeamMember(name="Louise Moncla", image="louise-moncla.jpg", link="http://bedford.io/team/louise-moncla/") }},
+    {{ TeamMember(name="Allison Black", image="allison-black.jpg", link="http://bedford.io/team/allison-black/") }},
+    {{ TeamMember(name="Sidney Bell", image="sidney-bell.jpg", link="http://bedford.io/team/sidney-bell/") }},
+    {{ TeamMember(name="Miguel Parades", image="miguel-parades.jpg", link="https://bedford.io/team/miguel-parades/") }},
+    {{ TeamMember(name="Colin Megill", image="colin-megill.jpg", link="http://www.colinmegill.com/") }},
+    {{ TeamMember(name="Barney Potter", image="barney-potter.jpg", link="http://bedford.io/team/barney-potter/") }},
+    {{ TeamMember(name="Pavel Sagulenko", image="pavel-sagulenko.jpg", link="https://neherlab.org/pavel-sagulenko.html") }},
+    {{ TeamMember(name="Charlton Callender", image="charlton-callender.jpg", link="http://bedford.io/team/charlton-callender/") }}
   </div>
 
 
   <div class="footer-small-text">
-    All <a href="https://github.com/nextstrain">source code</a> is freely available under the terms of the <a href="https://github.com/nextstrain/auspice/blob/master/LICENSE.txt">GNU Affero General Public License</a>. Screenshots may be used under a <a href="https://creativecommons.org/licenses/by/4.0/">CC-BY-4.0 license</a> and attribution to nextstrain.org must be provided.
+    All <a href="https://github.com/nextstrain">source code for Nextstrain</a> is freely available under the terms of an <a href="https://opensource.org/osd">open-source license</a>, typically <a href="https://opensource.org/licenses/AGPL-3.0">AGPL-3.0</a> or <a href="https://opensource.org/licenses/MIT">MIT</a>.
```

Comment:
Which of our projects are MIT? This doesn't appear on the main (nextstrain.org) footer

Reply:
ncov, ncov-ingest, nextclade, cli to name a few, plus some others.

The footer on nextstrain.org is also due for an update.